### PR TITLE
Added list group component support

### PR DESCRIPTION
**Pull request by bayaraajr**
## Brief description

Added list group component mentioned in #198 

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Sample pictures
image
image

...

## Further details

...

### CODE REVIEW

**Comment by Oli8** [comment]
Hey @bayaraajr, could you update the PR to target the develop branch instead of master, then rebase?

**Comment by bayaraajr** [comment]
Hello @Oli8, I've changed target branch to ```develop```.

**Comment by bayaraajr** [comment]
Is there anyone who is merging PRs to master or develop branch ? :)

File: src/components/_list-group.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      color: $white;
+      padding: 0.5rem;
+      padding-left: 1rem;
+
+      &:last-child {
+        border-bottom: 0;
+      }
+    }
+  }
+
```
Comment (Oli8) [inline src/components/_list-group.scss:54]:
```suggestion
```

**Review by Oli8 — APPROVED** [review]

**Pull request merged**